Ryujinx.HLE/HOS/Services/Settings/ISystemSettingsServer.cs

```diff
@@ -185,11 +187,16 @@ public byte[] GetFirmwareData(Switch device)
 
                 IFileSystem firmwareRomFs = firmwareContent.OpenFileSystem(NcaSectionType.Data, device.System.FsIntegrityCheckLevel);
 
-                IFile firmwareFile = firmwareRomFs.OpenFile("/file", OpenMode.Read);
+                Result rc = firmwareRomFs.OpenFile(out IFile firmwareFile, "/file", OpenMode.Read);
```

Comment:
```suggestion
                Result resultCode = firmwareRomFs.OpenFile(out IFile firmwareFile, "/file", OpenMode.Read);
```
and add an empty line after.